static/js/browser2/browser_page.tsx

```diff
@@ -53,6 +56,10 @@ export class BrowserPage extends React.Component<BrowserPagePropType> {
           ) : (
             <ArcSection dcid={this.props.dcid} nodeName={this.props.nodeName} />
           )}
+          {this.props.nodeType === nodeTypeEnum.CITY ||
+          this.props.nodeType == nodeTypeEnum.CENSUS_ZIPCODE_TABULATION_AREA ? (
```

Comment:
Would be nice to have a util function that centralize the logic of the rendering "components" given the node/page type.